install_config/http_proxies.adoc

```diff
@@ -68,7 +71,7 @@ leaving the external HTTP proxy as the only option.
 ----
 HTTP_PROXY=http://USERNAME:PASSWORD@10.0.1.1:8080/
 HTTPS_PROXY=https://USERNAME:PASSWORD@10.0.0.1:8080/
-NO_PROXY=master.hostname.example.com,172.30.123.45,docker-registry.default.svc.cluster.local
+NO_PROXY=master.hostname.example.com,10.1.0.0/16,172.30.0.0/16,docker-registry.default.svc.cluster.local
```

Comment:
This is under == Proxying Docker Pull == which is setting things for the docker environment. AFAIK that doesn't do anything. BTW, feel like duplicating this work in Docker upstream... that'd be awesome!